packages/application-shell/src/components/app-bar/app-bar.tsx

```diff
@@ -83,7 +83,12 @@ const AppBar = (props: Props) => {
           </Spacings.Inline>
         </Spacings.Inline>
       </div>
-      <div>
+      <div
+        css={css`
+          display: flex;
+        `}
+      >
+        <div id="extra-menu"></div>
```

Comment:
please suggest a better id or data-test name

Reply:
Also should it have a margin here or can it remain in mc-frontend instead where the portal is used?

Reply:
1. Naming

Given [here](https://github.com/commercetools/merchant-center-application-kit/pull/3066/files) on the prior PR we didn't introduce a `staff-bar--main` or something name dedicated to the usage we could maybe name this `left-of-profile`? Just as a suggestion?

2. Margin

I would not specify the margin here but define it in the MC FE. Maybe using spacer components.

Always found this a good article on the subject https://mxstbr.com/thoughts/margin/

Reply:
Agreed, no styles needed here.